src/chainstate/burn/db/sortdb.rs

```diff
@@ -702,6 +702,9 @@ const SORTITION_DB_SCHEMA_4: &'static [&'static str] = &[
     );"#,
 ];
 
+const SORTITION_DB_SCHEMA_5: &'static [&'static str] = &[r#"
+     DELETE FROM epochs;"#];
```

Comment:
So that they can be reinserted with the correct epochs.

Reply:
Hmm why was this deletion unnecessary to do for Epoch21? Is it because the end height of 2.05 was known at the time Epoch 2.05 was created (and this end height was the start height of epoch 2.1?

Reply:
It wasn't necessary for 2.1 because 2.1 required a sync from genesis (i.e., starting the node with a fresh chainstate), so the epochs were inserted during instantiation.

Reply:
Great, got it.